<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">#page{
border-image: url(:/_025fe56e-7f50-405f-b902-336be60dd152.jfif);
aligment: AlignHCenter;
}</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>180</y>
         <width>431</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="BotonJugar">
          <property name="text">
           <string>JUGAR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BotonSalir">
          <property name="text">
           <string>SALIR</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">#page_2{
border-image: url(:/_025fe56e-7f50-405f-b902-336be60dd152.jfif);
}</string>
      </property>
      <widget class="QWidget" name="Widget5Botones" native="true">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>190</y>
         <width>239</width>
         <height>249</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 255);</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>160</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="BotonBaja">
           <property name="text">
            <string>BAJA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BotonMedia">
           <property name="text">
            <string>MEDIA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BotonAlta">
           <property name="text">
            <string>ALTA</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="BotonAtras">
           <property name="text">
            <string>ATRAS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BotonContinuar">
           <property name="text">
            <string>CONTINUAR</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>seleccionar la dificultad</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
